pkg/universe.dagger.io/alpha/uffizzi/run.cue

```diff
@@ -0,0 +1,75 @@
+package uffizzi
+
+import (
+	"dagger.io/dagger"
+	"dagger.io/dagger/core"
+	"universe.dagger.io/docker"
+	"universe.dagger.io/bash"
+)
+
+#Run: {
+	source: dagger.#FS
+
+	uffizzi_user:     string
+	uffizzi_password: dagger.#Secret
+	uffizzi_project:  *null | string
```

Comment:
Why not optional? It's handled different by Dagger, [see this doc for more information](https://docs.dagger.io/1233/default-values-cue)

Reply:
Updated to optional field with values which make more sense. This was an oversight on my part. Thanks for checking.